elasticdl_client/common/k8s_client.py

```diff
@@ -47,6 +50,168 @@ def append_pod_ip_to_env(env):
     return env
 
 
+def try_get_class(module, name):
+    try:
+        cls = getattr(module, name)
+        return cls
+    except AttributeError:
+        return None
+
+
+def is_list_type(type_name):
+    if type_name.startswith("list["):
+        return True, type_name.split("[")[1].split("]")[0]
```

Comment:
From the method name `is_list_type `, I think this method only returns true | false instead of a tuple.

Reply:
Renamed.